src/components/UI/Modal/Modal.module.scss

```diff
@@ -0,0 +1,36 @@
+.modal {
+    position: fixed;
+    top: 0;
+    left: 0;
+    height: 100vw;
+    min-width: 50%;
+    z-index: 2;
+}
+
+.modalMain {
+    margin-top: 50%;
+    margin-left: 25%;
+    background: white;
+    box-shadow: 5px 10px 18px rgba(0, 0, 0, 0.05);
+    border-radius: 30px;
+    height: auto;
+}
+.closeButton {
+    border: none;
+    background: none;
+    width: auto;
+    margin-top: 16px;
+    margin-right: 16px;
+    font-size: 18px;
+    text-align: end;
+    &:hover {
+        font-size: 20px;
```

Comment:
on hover increasing font-size. Slightly push down Send an invite to multiple friends. I suggest removing this.

Reply:
@ArnasDickus Regarding to have just one active modal form at the time, to make that possible probably I'll have to use redux (global state). That way I  can also hide this form when clicking on any other navbar link